plot.py

```diff
@@ -515,10 +515,10 @@ def compute_target(target, degradation_model, cell_id, sign_change, mode, averag
 
         if mode == "cc":
             yrange = v_range
```

Comment:
I don't understand where this comes from.

Reply:
Neither do I. I have not got this this file yet. Maybe I need a rebase...